diff --git a/sponsorblock.lua b/sponsorblock.lua
index 4a8fc40..78da6b6 100644
--- a/sponsorblock.lua
+++ b/sponsorblock.lua
@@ -21,7 +21,7 @@ local options = {

     -- Note that sponsored segments may ocasionally be inaccurate if this is turned off
     -- see https://blog.ajay.app/voting-and-pseudo-randomness-or-sponsorblock-or-youtube-sponsorship-segment-blocker
-    local_database = true,
+    local_database = false,

     -- Update database on first run, does nothing if local_database is false
     auto_update = true,
@@ -37,7 +37,7 @@ local options = {
     display_name = "",

     -- Tell the server when a skip happens
-    report_views = true,
+    report_views = false,

     -- Auto upvote skipped sponsors
     auto_upvote = false,
@@ -91,9 +91,9 @@ end
 local utils = require "mp.utils"
 scripts_dir = mp.find_config_file("scripts")

-local sponsorblock = utils.join_path(scripts_dir, "sponsorblock_shared/sponsorblock.py")
-local uid_path = utils.join_path(scripts_dir, "sponsorblock_shared/sponsorblock.txt")
-local database_file = options.local_database and utils.join_path(scripts_dir, "sponsorblock_shared/sponsorblock.db") or ""
+local sponsorblock = utils.join_path(scripts_dir, "../sponsorblock/sponsorblock_shared/sponsorblock.py")
+local uid_path = utils.join_path(scripts_dir, "../sponsorblock/sponsorblock_shared/sponsorblock.txt")
+local database_file = options.local_database and utils.join_path(scripts_dir, "../sponsorblock/sponsorblock_shared/sponsorblock.db") or ""
 local youtube_id = nil
 local ranges = {}
 local init = false
@@ -553,10 +553,10 @@ function submit_segment(category)
 end

 mp.register_event("file-loaded", file_loaded)
-mp.add_key_binding("g", "set_segment", set_segment)
-mp.add_key_binding("G", "submit_segment", submit_segment)
-mp.add_key_binding("h", "upvote_segment", function() return vote("1") end)
-mp.add_key_binding("H", "downvote_segment", function() return vote("0") end)
+mp.add_key_binding(nil, "set_segment", set_segment)
+mp.add_key_binding(nil, "submit_segment", submit_segment)
+mp.add_key_binding(nil, "upvote_segment", function() return vote("1") end)
+mp.add_key_binding(nil, "downvote_segment", function() return vote("0") end)
 -- Bindings below are for backwards compatibility and could be removed at any time
 mp.add_key_binding(nil, "sponsorblock_set_segment", set_segment)
 mp.add_key_binding(nil, "sponsorblock_submit_segment", submit_segment)
